superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -0,0 +1,196 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import styled from '@superset-ui/style';
+import Icon from 'src/components/Icon';
+import { Card } from 'src/common/components';
+
+const MenuIcon = styled(Icon)`
+  width: 14px;
+  height: 14px;
+  position: relative;
+  top: 1px;
+`;
+
+const ActionsWrapper = styled.div`
+  width: 64px;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const StyledCard = styled(Card)`
+  width: 459px;
+
+  .ant-card-body {
+    padding: 16px 8px;
+  }
+  .ant-card-meta-detail > div:not(:last-child) {
+    margin-bottom: 0;
+  }
+`;
+
+const Cover = styled.div`
+  height: 264px;
+  overflow: hidden;
+
+  .cover-footer {
+    transform: translateY(36px);
+    transition: 0.2s ease-out;
+  }
+
+  &:hover {
+    .cover-footer {
+      transform: translateY(0);
+    }
+  }
+`;
+
+const GradientContainer = styled.div`
+  position: relative;
+  display: inline-block;
+
+  &:after {
+    content: '';
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    display: inline-block;
+    background: linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 47.83%,
```

Comment:
Hmmmmmmmm.... perhaps outside the scope of this PR, but it'd be nice to find a way to address rgba colors like this on theme variables with a SASS mixin or something, so we can have it do the right thing if we switched to dark mode or something.

Reply:
I guess we could just set transparency stops with two more hex characters, to build hex alpha colors? I.e. 

```
background: linear-gradient(
      180deg,
      ${({ theme }) => theme.colors.grayscale.dark2}00 47.83%,
      ${({ theme }) => theme.colors.grayscale.dark2}00 79.64%,
      ${({ theme }) => theme.colors.grayscale.dark2}${({ theme }) => theme.opacities.medium} 100%,
 100%

``` ,

Reply:
I tried this, eslint or prettier is refusing to keep `theme.colors.grayscale.dark2}${({ theme }) => theme.opacities.medium} 100%` on the same line

Reply:
also, opacities currently include `%`

Reply:
Ahh yes... we'd need to add hex-based opacities... which seems like they'd be used most if we're proceeding with hex colors.

Reply:
Tracking as a new issue, assigned to me... will tackle it later.
https://github.com/apache/incubator-superset/issues/10598